common/utils/ga.ts

```diff
@@ -1,7 +1,7 @@
 import ReactGA, { EventArgs as GaEvent } from 'react-ga';
 
 // TODO this wrapper function is redundant but pervasive; consider getting rid of it
```

Comment:
Could we just do away with this wrapper function entirely?

Reply:
This is an older TODO that I'd consider dealing with as part of whatever is happening with GTM instead. 
@jamieparkinson , was the "consider" because it could still be useful/allowed typing?

Reply:
I think I might've just written that to make it sound like I had a better reason than "I don't want to do that now"

Reply:
This is my favourite honest comment I've seen in a while 😂 Am I wrong in thinking this'll all be refactored when we implement GTM anyway?

Reply:
Not actually sure but that alone is reason to wait and see, fine to regard it as out of scope for this